app/request_list.tsx

```diff
@@ -2,11 +2,41 @@
 import * as React from "react";
 import ReactTable from "react-table";
 import * as matchSorter from "match-sorter";
+import * as $ from "jquery";
 export interface RequestObj {
   requests: Array<chrome.webRequest.WebRequestDetails>;
   handleIntercept: React.MouseEventHandler<HTMLButtonElement>;
 }
 const RequestList = (props: RequestObj) => {
+  const InterceptComponent = props => {
+    let responseText = "{msg:hello}";
```

Comment:
I have hardcoded the text here on L12. Since this component is a Stateless component, I would not be able to access its state or refs and hence cannot record the `event.target.value` of the textarea to be passed on to props.handleIntercept when the Intercept button is clicked. Should I make it a stateful component to achieve this?